docs/pages/choose-an-edition/teleport-enterprise/aws-kms.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: AWS KMS
+description: Configure Teleport to store CA private keys in the AWS Key Management Service
+h1: Store Teleport Private Keys in AWS KMS
+---
+
+This guide will show you how to set up your Teleport Cluster to use the AWS Key
+Management Service (KMS) to store and handle the CA private key material used to
+sign all certificates issued by your Teleport cluster.
+
+Teleport generates private key material for its internal Certificate Authorities
+(CAs) during the first Auth Server's initial startup.
+These CAs are used to sign all certificates issued to clients and hosts in the
+Teleport cluster.
+When configured to use AWS KMS, all private key material for these CAs
+will be generated, stored, and used for signing inside of AWS KMS.
+Instead of the actual private key, Teleport will only store the ID of the KMS
+key.
+In short, private key material will never leave AWS KMS.
+
+If launching a new Teleport cluster this will all be handled during initial
+startup with no specific interventions required after configuration.
+For existing Teleport clusters that already have private key material on disk,
```

Comment:
I suspect most people interested in this feature will be those using Teleport on AWS with DynamoDB, so the "already on disk" point isn't entirely accurate..